### PR TITLE
PA-5696 - Add support of Speeding Violations API endpoint

### PR DESCRIPTION
**Pull request by amatkivskiy**

### CODE REVIEW

File: mojio-sdk-rest/src/test/java/io/moj/java/sdk/rest/GetTripSpeedingViolationTest.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+import okhttp3.mockwebserver.MockWebServer;
+import okio.Buffer;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import retrofit2.Response;
+
+import java.io.IOException;
+import java.net.URL;
```
Comment (mtauzhnianskyi) [inline mojio-sdk-rest/src/test/java/io/moj/java/sdk/rest/GetTripSpeedingViolationTest.java]: import optimization is suggested

Comment (amatkivskiy) [reply mojio-sdk-rest/src/test/java/io/moj/java/sdk/rest/GetTripSpeedingViolationTest.java]: Fixed in d80f013

**Comment by amatkivskiy** [comment]
@mtauzhnianskyi Should be ready fo review.

**Comment by amatkivskiy** [comment]
@dacheng81 We discovered with @mtauzhnianskyi  that sometimes `GetTripSpeedingViolationTest.java`  test fails because of 
```
java.lang.IllegalArgumentException: in == null

	at okio.Buffer.readFrom(Buffer.java:246)
	at okio.Buffer.readFrom(Buffer.java:234)
	at io.moj.java.sdk.test.utils.MockWebServerUtils.successFromFile(MockWebServerUtils.java:10)
	at io.moj.java.sdk.rest.GetTripSpeedingViolationTest.get_trip_speeding_violation_test(GetTripSpeedingViolationTest.java:32)
``` 
But most of the times it passes successfully. I found that issue is related to wrong project import by Android Studio. If someone faces this issue just re-import the project.

**Pull request merged**